layouts/partner/aws.html

```diff
@@ -8,217 +8,123 @@
-                </a>
+            <div class="flex items-center flex-wrap justify-center my-4">
+                {{ range .Params.customers }}
+                    {{ if .link }}
+                        <a data-track="customer-logo" href="{{ .link }}" target="_blank" rel="noreferrer noopener">
+                            <img class="m-4 h-12" src="{{ .logo }}" alt="{{ .name }}" title="{{ .name }}" />
+                        </a>
+                    {{ else }}
+                        <img class="m-4 h-12" src="{{ .logo }}" alt="{{ .name }}" title="{{ .name }}" />
+                    {{ end }}
+                {{ end }}
             </div>
         </div>
         <p class="text-blue-500 text-center font-bold mt-10">
             <a class="link" data-track="logos-case-study" href="/case-studies/">Read our customer use cases</a> &rarr;
         </p>
+        <img class="w-32 mx-auto" src="/images/partners/aws-devops-badge.png" alt="AWS Devops Badge" />
     </section>
 
     <!-- Automation API -->
     <section id="automation-api" class="container mx-auto mb-12 px-8 lg:px-0">
-        <div class="flex flex-col lg:flex-row">
-            <div class="w-full lg:w-5/12 px-8">
-                <h3>Programmatic Infrastructure as Code</h3>
-                <p>
-                    Pulumi <a href="/automation/">Automation API</a> exposes the full power of infrastructure as code through a programmatic interface, instead of
-                    through CLI commands. Automation API lets you use the Pulumi engine as an SDK, enabling you to create software that can create, update, configure, and destroy
-                    infrastructure dynamically. This enables you to build custom cloud interfaces that are tailored to your team, organization, or customers.
-                </p>
-                {{ partial "sparkle-arrow-cta" (dict "url" "/blog/automation-api" "text" "Learn More About Automation API" "textColor" "grey" "textAlign" "left") }}
-            </div>
+        {{ with .Params.automation_api }}
+            <div class="flex flex-col lg:flex-row">
+                <div class="w-full lg:w-5/12 px-8">
+                    <h3>{{ .header }}</h3>
+                    <p>{{ .body | markdownify}}</p>
+                    {{ partial "sparkle-arrow-cta" (dict "url" .link "text" .cta "textColor" "grey" "textAlign" "left") }}
+                </div>
 
-            <div class="w-full mt-8 lg:mt-0 lg:w-7/12">
-                <img class="shadow-xl rounded" src="https://www.pulumi.com/uploads/content/blog/automation-api/http.gif" alt="Pulumi over HTTP" />
-                <!-- {{ partial "code" (dict "code" .Params.automation_api_code "lang" "js" "mode" "dark" "chrome" true) }} -->
+                <div class="w-full mt-8 lg:mt-0 lg:w-7/12">
+                    <img class="shadow-xl rounded" src="https://www.pulumi.com/uploads/content/blog/automation-api/http.gif" alt="Pulumi over HTTP" />
```

Comment:
there needs to be some white space above this gif. Will need to replace the gif as well. What dimension should this be?

Reply:
896px height by 447px width